File: scala/scala-impl/test/org/jetbrains/plugins/scala/failed/parser/FailedParserTest.java
package org.jetbrains.plugins.scala.failed.parser;

import junit.framework.Test;
import org.jetbrains.plugins.scala.base.ScalaFileSetTestCase;
import org.junit.runner.RunWith;
import org.junit.runners.AllTests;

/**
 * @author Nikolay.Tropin
 */
@RunWith(AllTests.class)
public abstract class FailedParserTest extends ScalaFileSetTestCase {

    public FailedParserTest() {
        super("/parser/failed");
    }

    public static Test suite() {
        return new ScalaFailedParserTest();
    }

    @Override
    protected boolean shouldPass() {
        return false;
    }
}

File: scala/scala-impl/test/org/jetbrains/plugins/scala/lang/parser/FailedParserTest.scala
package org.jetbrains.plugins.scala
package lang
package parser

import org.jetbrains.plugins.scala.base.{DefaultFileSetTestTransform, NoSdkFileSetTestBase}

import java.nio.file.Path

class FailedParserTest extends NoSdkFileSetTestBase with DefaultFileSetTestTransform {
  override protected def relativeTestDataPath: Path = Path.of("parser", "failed")

  override protected def shouldPass: Boolean = false
}
